### PR TITLE
add HD_SUCCESS func

### LINKED ISSUE #4: Add customizable SUCCESS text line for CLI
Note the following points.
- The base color for denoting texts is suggested to be `GREEN`.
- Use the ASCII stroke tick sign or `[SUCCESS]` text to be the symbol.
- Make sure the typeface on the CLI is in `BOLD` style.

### PR DESCRIPTION
**Pull request by 0vj**

### CODE REVIEW

File: spectrum.go [hunk 1]
```diff
@@ -48,3 +52,5 @@ func TX_BOLD(textobjc string) string { return "\u001b[1m" + textobjc + "\u001b[0
 func TX_UNDERLINE(textobjc string) string { return "\u001b[4m" + textobjc + "\u001b[0m" }
 
 func TX_INVERTED(textobjc string) string { return "\u001b[7m" + textobjc + "\u001b[0m" }
+
+func HD_SUCCESS(textobjc string) string { return FR_GREEN(TX_BOLD("[SUCCESS] " + strings.ToUpper(textobjc))) }
```
Comment (gridhead) [inline spectrum.go]: We do not need the entire success message to be in caps. Only the `SUCCESS` part will do.

**Review by gridhead — CHANGES_REQUESTED** [review]
Please check the changes I have suggested and follow through accordingly.

File: spectrum.go (hunk 1 above)
Comment (0vj) [reply spectrum.go]: I fixed it !

File: spectrum.go [hunk 2]
```diff
@@ -48,3 +48,5 @@ func TX_BOLD(textobjc string) string { return "\u001b[1m" + textobjc + "\u001b[0
 func TX_UNDERLINE(textobjc string) string { return "\u001b[4m" + textobjc + "\u001b[0m" }
 
 func TX_INVERTED(textobjc string) string { return "\u001b[7m" + textobjc + "\u001b[0m" }
+
+func HD_SUCCESS(textobjc string) string { return FR_GREEN(TX_BOLD("[SUCCESS] " + textobjc)) }
```
Comment (gridhead) [inline spectrum.go:52]: Much better. :+1:

**Review by gridhead — APPROVED** [review]
LGTM. Merging now.

**Pull request merged**